src/main/java/brave/opentracing/BraveTracer.java

```diff
@@ -52,7 +54,7 @@
  * @see BraveSpan
  * @see Propagation
  */
-public final class BraveTracer implements Tracer {
+public final class BraveTracer extends ThreadLocalActiveSpanSource implements Tracer {
```

Comment:
Brave already has its own active span source. There should be a wrapper around the brave abstraction so that spans created in OT are accessible in brave and the other way around